.travis.yml

```diff
@@ -20,6 +20,7 @@ addons:
     - libxi-dev
 node_js:
   - 6.9.1
+  - 8
```

Comment:
what's this?

Reply:
Adding another set of tests on Travis for node v8

Reply:
Which node versions Travis CI should test on. 6 and 8 are the latest LTS (long-term support) versions, so good to test on those.

Reply:
If you click on the travis CI link below you can see there are multiple test runs.